src/collective/fingerpointing/browser/configure.zcml

```diff
@@ -20,6 +20,7 @@
       name="fingerpointing-audit-log"
       for="Products.CMFPlone.interfaces.IPloneSiteRoot"
       class="collective.fingerpointing.browser.logview.LogView"
+      template="logview.pt"
```

Comment:
is there an advantage on setting the template here?

Reply:
Its the common documented way, its used all over the place. Also it makes reusing of the view easier, like using it with a different template.